store/src/main/java/com/issoft/store/RandomStorePopulator.java

```diff
@@ -101,6 +102,10 @@ public void prettyProducts(List<Product> category) {
         }
     }
 
-
+    public List<Product> createOrder(List<Product> products){
+        List<Product> selectedProducts = products.subList(3, 5);
+        return selectedProducts;
+    }
 }
```

Comment:
I am not sure that this method will be required in case if product selection by user will be implemented.